kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -119,6 +121,11 @@ def _open_merge_request(
         else:
             body = DEFAULT_PR_BODY
 
+        body = (
+            f"# Automatic Update of {self.runtime_environment} runtime-environment\n"
```

Comment:
should we add `overlays/{runtime_envrionment_name}`?

Reply:
if it is happening in an overlay that is a good information

Reply:
Yeah, this can be added